target/linux/ath79/image/generic.mk

```diff
@@ -128,6 +128,15 @@ define Device/buffalo_wzr-hp-g450h
 endef
 TARGET_DEVICES += buffalo_wzr-hp-g450h
 
+define Device/comfast_cf-e110n
+  ATH_SOC := qca9533
+  DEVICE_TITLE := COMFAST CF-E110N
+  DEVICE_PACKAGES := rssileds kmod-leds-gpio -swconfig -uboot-envtools
```

Comment:
In the description you're saying:

> The CF-E110N is very similar to the CF-E320N V2

Looking at `cf-e320n-v2` in `ar71xx` I can see `DEVICE_PACKAGES += kmod-usb-core kmod-usb2` is there any reason why you don't include them also?

Reply:
The CF-E320N-V2 is based on the QCA9531 SoC and the CF-E110N is based on the QCA9533, which I read (https://forum.archive.openwrt.org/viewtopic.php?id=69479) it has no USB. Therefore I thought it was not needed.

Reply:
Ok, I'm not sure, but I think, that I've seen some USB nodes in this DTS, that's why I was confused.